gnucash/gnucash-locale-windows.c

```diff
@@ -24,17 +25,15 @@
 #include <Windows.h>
 #include <fcntl.h>
 #include <glib/gi18n.h>
-
-//sacrificial prototype
-void set_win32_thread_locale(char **sys_locale);
+#include "gnucash-locale-platform.h"
 
 /* If one of the Unix locale variables LC_ALL, LC_MESSAGES, or LANG is
  * set in the environment check to see if it's a valid locale and if
  * it is set both the Windows and POSIX locales to that. If not
  * retrieve the Windows locale and set POSIX to match.
  */
-void
-set_win32_thread_locale(char **sys_locale)
+char *void
```

Comment:
Is that a typo? Function returning `char* void`?

Reply:
Done in a followup-commit.